src/hooks/useAppBootstrap.js

```diff
@@ -140,7 +140,11 @@ export const useAppBootstrap = client => {
 
         setInitialRoute({
           stack: isHome ? fallback : undefined,
-          root: !isHome ? fallback : undefined
+          root: isConnected
+            ? !isHome
+              ? fallback
+              : undefined
+            : strings.errorScreens.offline
```

Comment:
I don't understand why this is needed (and also why it isn't needed for the `stack` prop). Can you describe a scenario in the commit message?

Reply:
This is once more an issue with app bootstrap complexity. This provides us initial error param as offline, whereas before it was handled with a race condition on the handleOffline() function. My main objective in terms of refactoring is to tackle this file and make initial routes understandable.